examples/ccsds-frames-sdls/src/main/yamcs/etc/yamcs.ccsds-frames-sdls.yaml

```diff
@@ -11,7 +11,7 @@ services:
         - tm_dump
   - class: org.yamcs.parameter.SystemParametersService
     args:
-      producers: ['jvm', 'fs']
```

Comment:
why this change?

Reply:
As agreed, we keep the formatting changes for consistency